# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../lesstif/c++.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- lesstif-0.94.4/include/Motif-2.1/Xm/VendorSP.h.vanilla	2007-02-28 20:02:49.690434500 +0000
+++ lesstif-0.94.4/include/Motif-2.1/Xm/VendorSP.h	2007-02-28 20:20:09.431668250 +0000
@@ -28,12 +28,13 @@
 
 #include <Xm/XmP.h>
 #include <Xm/VendorS.h>
-#include <X11/ShellP.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#include <X11/ShellP.h>
+
 XMLIBEXPORT extern Cardinal _XmFilterResources(XtResource *resources,
 				   Cardinal numResources,
 				   WidgetClass filterClass,

